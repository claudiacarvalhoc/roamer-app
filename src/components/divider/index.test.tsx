import React from 'react';
import Divider, { DividerProps } from './index';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

const getShallowWrapper = (props) => {
    return shallow(<Divider {...props} />);
};

describe('<Divider />', () => {
    it('render correctly', () => {
        const props: DividerProps = {
            className: 'test',
        };
        const component = getShallowWrapper(props);
        expect(toJson(component)).toMatchSnapshot();
    });
});
