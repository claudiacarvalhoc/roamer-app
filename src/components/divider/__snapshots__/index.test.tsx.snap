// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Divider /> render correctly 1`] = `
<div
  className="test divider"
/>
`;
